### PR TITLE
Spanish Translation of ch06-02 (line 1 to 179)

### PR DESCRIPTION
**Pull request by dbucci24**
I have done a partial translation of this file from line 1 to 179 (1047 words)

### CODE REVIEW

File: second-edition/src/ch06-02-match.md [hunk 1]
```diff
@@ -37,36 +36,36 @@ fn value_in_cents(coin: Coin) -> u32 {
...
+gran diferencia: con `if`, la expresión necesita devolver un valor Booleano.
+Aquí, puede ser de cualquier tipo. El tipo de `coin` en este ejemplo es la enumeración
+`Coin` que es definida en el Listado 6-3.
+
+A continuación están los brazos del `match`. Un brazo tiene dos partes: un patrón
+y algún código. El primer brazo aquí tiene un patrón que es el valor `Coin::Penny`
+y luego el operador `=>` que separa el patrón y el código a ejecutar. El código en este caso
+es solo el valor 1 `1`. Cada brazo está separado del próximo con una coma.
+
+Cuando la expresión `match` ejectua, compara el valor resultante contra el
```
Comment (thecodix) [inline second-edition/src/ch06-02-match.md]: *ejecuta

File: second-edition/src/ch06-02-match.md [hunk 2]
```diff
@@ -37,36 +36,36 @@ fn value_in_cents(coin: Coin) -> u32 {
...
+entera `match`.
+
+Las llaves tipicamente no son usadas si el código del brazo de coincidencia es corto, como
+lo es en el Listado 6-3 donde cada brazo solo retorna un valor. Si quieres ejecutar múltiples
+líneas de código en un brazo de coincidencia, puedes usar llaves. Por ejemplo, el
+siguiente código imprimirá “Lucky penny!” cada vez que el método sea llamado
+con un `Coin::Penny` pero seguirá retornando el último valor del bloque, `1`:
 
 ```rust
 # enum Coin {
```
Comment (thecodix) [inline second-edition/src/ch06-02-match.md]: You may want to translate the words here

eg: Coin -> Moneda

File: second-edition/src/ch06-02-match.md [hunk 3]
```diff
@@ -118,18 +117,18 @@ enum Coin {
...
-pattern that matches values of the variant `Coin::Quarter`. When a
-`Coin::Quarter` matches, the `state` variable will bind to the value of that
-quarter’s state. Then we can use `state` in the code for that arm, like so:
+En la expresión de coincidencia para este código, agregamos una variable llamada `state` al
+patrón que coincide con el valor de la variante `Coin::Quarter`. Cuando un
+`Coin::Quarter` coincide, la variable `state` se juntará al valor del 
+estado de alojamientos. Entonces podemos usar `state` en el código para ese brazo, asi:
 
 ```rust
 # #[derive(Debug)]
```
Comment (thecodix) [inline second-edition/src/ch06-02-match.md]: Again, please try and translate the variables used here, included UsState (maybe EstadosUs)

Comment (dbucci24) [reply second-edition/src/ch06-02-match.md]: fixed, look at it now, to see if it's okay

**Comment by thecodix** [comment]
machine translated

**Comment by dbucci24** [comment]
I will work on it :)

**Pull request merged**